docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -683,6 +683,123 @@ Where adherence score = 0 for primary or secondary nonadherent; and adherence sc
     - Burn in period will allow the distribution of follow-up appointments to reach equilibrium prior to time start 
 
 
+Creation of "Untreated" SBP Values on Initializaiton
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+GBD values for SBP which are used in this sim reflect the US distribution of SBP **including** medication benefits. 
+Therefore, by later applying treatment benefits to certain simulants, we are double counting the population level 
+benefit of treatment. To avoid this, we must add SBP at initialization to create an "untreated" SBP level. 
```

Comment:
Nit: I know this is already merged, but it might be clearer to say "To avoid this, we must add SBP to all adherent simulants on medication upon initialization in order to create an "untreated" baseline SBP level."